backend/services/email.py

```diff
@@ -45,14 +45,14 @@ def send_battle_result(battle, creator_pokemon_list, opponent_pokemon_list):
             "battle_winner": battle.winner.email.split("@")[0],
             "battle_id": battle.id,
             "creator_team": [
```

Comment:
This could also become a list comprehension: `pokemon.name for pokemon in creator_pokemon_list`